nservicebus/azure/data-distribution_content_[5.2,6).partial.md

```diff
@@ -0,0 +1,7 @@
+snippet: UniqueQueuePerEndpointInstance
```

Comment:
Let me double check, I think it might not, we could either re-work the content here or raise PR to the DocsEngine.